### PR TITLE
gftools-add-designer

### PR DESCRIPTION
**Pull request by m4rc1e**
Wipasaurus.

### CODE REVIEW

**Comment by m4rc1e** [comment]
@RosaWagner @vv-monsalve feel free to review.

**Review by m4rc1e — COMMENTED** [review]
there's a missing dependency

```
Traceback (most recent call last):
  File "/Users/marcfoley/Type/tools/venv3/bin/gftools-add-designer.py", line 34, in <module>
    from gftools.designers_pb2 import DesignerInfoProto
ModuleNotFoundError: No module named 'gftools.designers_pb2'
```

File: bin/gftools-add-designer.py [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+In order to use the spreadsheet command in this script, you will need to
+download the folder which contains the spreadsheet and designer images.
+You can then use the ``--spreadsheet`` arg.
+
+Usage:
+# Add or update a designer entry. User will need to hand complete the bio.html
+$ gftools add-designer ~/Type/fonts/catalog/designers "Theo Salvadore" path/to/img.png
+
+# Add or update a designer entry and use the spreadsheet to create the bio.html file
```
Comment (m4rc1e) [inline bin/gftools-add-designer.py:28]: Change "Add or update a designer entry and use the spreadsheet to create the bio.html file" to "Add or update a designer entry using the spreadsheet."

File: bin/gftools-add-designer.py [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+There is also an image directory where the designer's profile pics are uploaded into.
+
+In order to use the spreadsheet command in this script, you will need to
+download the folder which contains the spreadsheet and designer images.
+You can then use the ``--spreadsheet`` arg.
+
+Usage:
+# Add or update a designer entry. User will need to hand complete the bio.html
+$ gftools add-designer ~/Type/fonts/catalog/designers "Theo Salvadore" path/to/img.png
```
Comment (RosaWagner) [inline bin/gftools-add-designer.py:26]: We want --img_path as a optional argument.

Comment (RosaWagner) [reply bin/gftools-add-designer.py:26]: ~/Type/fonts/catalog/designers --> path/to/local/clone/fonts/catalog/designers ?

**Review by m4rc1e — COMMENTED** [review]
If a designer directory already exists and the image filename is updated, the old image isn't removed.

**Review by m4rc1e — COMMENTED** [review]
For social media links in the bio.html file, the text should only be "twitter" etc. Not "twitter/username".

File: bin/gftools-add-designer.py (hunk 2 above)
Comment (m4rc1e) [reply bin/gftools-add-designer.py:26]: I'll implement optional image paths in v2.

I agree with the path example.

**Comment by m4rc1e** [comment]
Alright, I've added the following:
- Fixes for social media links
- You don't need to specify an image in order to run the tool
- Remove older images

@RosaWagner please review

**Comment by RosaWagner** [comment]
Works like a charm.

**Comment by m4rc1e** [comment]
Thanks. Let's merge this and cut a release.

**Pull request merged**